consensus/src/quorum_store/batch_generator.rs

```diff
@@ -250,6 +251,9 @@ impl BatchGenerator {
         } else {
             counters::PULLED_TXNS_COUNT.inc();
             counters::PULLED_TXNS_NUM.observe(pulled_txns.len() as f64);
+            if pulled_txns.len() as u64 == max_count {
+                counters::BATCH_PULL_FULL_TXNS.observe(max_count as f64)
```

Comment:
why not set this check as a boolean label to `counters::PULLED_TXNS_COUNT`? is max_count not available else where?

Reply:
Yeah, this is actually the first time putting max_count in metrics.